tests/unit/test_connector.py

```diff
@@ -157,3 +157,16 @@ async def test_create_async_connector() -> None:
     connector = await create_async_connector()
     assert connector._loop == asyncio.get_running_loop()
     await connector.close_async()
+
+
+def test_Connector_close_kills_threads() -> None:
+    """Test that Connector.close kills background threads."""
+    # several fixtures use threads so get count prior to test
+    prior_threads = threading.active_count()
+    # open and close 10 Connector objects
+    for _ in range(10):
```

Comment:
Is it possible to access the thread directly and call `is_alive()` after the close is called? This seems like a more reliable way to determine if it's still running or not

Reply:
Done